### PR TITLE
[FR] Update swagger reference

### PR DESCRIPTION
**Pull request by maririos**
Now that the service team has merged their 2.1-preview.2 swagger, use that one instead of a local copy.

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/Language.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ public Language(string value)
         private const string ItValue = "it";
         private const string NlValue = "nl";
         private const string PtValue = "pt";
+        private const string JaValue = "ja";
```
Comment (kinelski) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/Language.cs:32]: Just out of curiosity, is this Japanese?

**Review by kinelski — APPROVED** [review]
LGTM! すばらしい！

**Review by kristapratico — APPROVED** [review]
yay!

File: sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/Language.cs (hunk 1 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/Language.cs:32]: It is!

**Pull request merged**